src/spyglass/position/v1/position_dlc_project.py

```diff
@@ -308,9 +308,10 @@ def insert_new_project(
                 raise ValueError(
                     f"bodypart: {bodypart} not found in BodyPart table"
                 )
-        kwargs.update({"numframes2pick": frames_per_video, "dotsize": 3})
+        kwargs_copy = kwargs.copy()
```

Comment:
It's probably better to use deepcopy here if you don't want to change the original